courses/gnatcheck/labs/coding_standard.rules

```diff
@@ -5,108 +5,108 @@
 --  11.1.2 OBJECT ORIENTATION
 --  -------------------------
 
-+R Deep_Inheritance_Hierarchies : 5
++RDeep_Inheritance_Hierarchies:5
```

Comment:
It was clearer with the original indent, why change it?

Reply:
it seems to be the case on the whole file, some "pretty" printer at play there?

Reply:
The problem is that GNAT Studio does not handle spaces in the rules!
While the file will still work, the GUI menu does not show any of those rules as part of the file, which can really mess things up. So I wanted to be consistent (because in class I expect most people to use the GUI)